### DISCUSSION TITLE
Relation Manager and One-to-one Relationship - Display Form Instead of Table

### DISCUSSION THREAD

**Discussion by myregistration** [filamentphp/filament#12169] [Help] [answered]
### Package

Other

### Package Version

v3

### How can we help you?

Is it possible to display the related form and not a table for a one-to-one relationship on the Relation Manager page?  Displaying a table of one record doesn't make sense so I just want to display the relative form instead.   For example, I would like to display a Profile form on the User Resource page.  Thanks!

**Comment by leandrocfe** [comment, marked as answer]
Use Layout components  + `relationship()`-  https://filamentphp.com/docs/3.x/forms/layout/getting-started

```php
Section::make('profile')
    ->relationship('profile') //profile is the relationship name
    ->schema([
        // ...
    ])
```

**Reply by myregistration** [reply]
That worked great!  Is there a easy way to have them separate on the same page with submit buttons for each?   I'm learning so can do some research, but if you have a link would be appreciated.  Thanks again!

**Reply by Alex-S-Davies** [reply]
I use [subresource navigation](https://filamentphp.com/docs/3.x/panels/resources/viewing-records#adding-view-pages-to-resource-sub-navigation) for this, don't forget to add a new Page to `getPages()`

**Reply by leandrocfe** [reply]
hum.. these fields should exist in the same layout component..
You can also use tabs - https://filamentphp.com/docs/3.x/forms/layout/tabs
